grants/search.js

```diff
@@ -34,6 +46,15 @@ function isPostcode(input) {
     );
 }
 
+/**
+ * Turns a number into a localised count
+ * eg. 123456 => 123,456
+ * @param {string} str
+ */
+function numberWithCommas(str = '') {
```

Comment:
Maybe boop this into a lib file and copy over the test for this too?

Reply:
Thinking about it, not sure what the tradeoffs of using `Number#toLocaleString` here?

Reply:
None at all – will swap it out.